core/Enclave/enclave_hsm.edl

```diff
@@ -129,6 +129,10 @@ enclave {
         public sgx_status_t enclave_get_rand([out, size=datalen] uint8_t *data,
                                                 uint32_t datalen);
 
+        public sgx_status_t enclave_verify_policy([in, size=quote_len] uint8_t* quote, uint32_t quote_len, 
+                            [in, size=mr_signer_good_len] const char* mr_signer_good, uint32_t mr_signer_good_len, 
```

Comment:
please keep the same format with other API

Reply:
I don't understand what you mean. Could you say it in more detail

Reply:
keep alignment

Reply:
Ok, I will change it.